Users/coafile.rst

```diff
@@ -39,6 +39,15 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+It can be used to define the type of files you usually
+don't want to lint like minified files
+and backup files (e.g. ``*.min.js``, ``*.orig``)
```

Comment:
This can be a bit misleading. I suggest you write it in the form minified files (``*.min.js``) and backup files (``*.orig``).